Sprint 2

miguelmballeza opened the pull request:
Entrega del Segundo Sprint.

miguelmballeza committed:
Commit de Referencia para la Rama de sprint_2

Este commit fue usado de referencia para la rama sprint_2.

miguelmballeza committed:
Se genero la estructura básica para un proyecto de Node.js con Express.

Se genero la estructura básica que se necesita para un proyecto en
Node.js con el Framework Express además de los archivos JSON
correspondientes, el entry point cuenta con las instrucciones necesarias para
el levantamiento del Servidor en el puerto 3000 además de una callback
sencilla de impresión en consola si el servidor esta levantado o no.
Además de el levantamiento de las 5 principales rutas para poder trabajar
en la carpeta "views" sus respectivos doc HTML y en la subcarpeta CSS sus
respectivas hojas deestilo.
Y finalmente se avanzo un poco en relación al navbar de la
página, asi como la propia página de inicio, se necesita terminar de
darle estilos a la sección principal además del resto de la página.

miguelmballeza committed:
Modificación de README.md y Creación de retro.md

Se modifico README.md para agregar el link del tablero de trabajo de
nuestro Trello. Y se creo el archivo retro.md con las conclusiones de la
ceremonia de Retrospectiva para nuestra mejora continua.

Trxpxrx committed:
se agrego la parte del carrito con sus iconos, imagenes y hoja de estilo

Trxpxrx committed:
Se corrigio la ruta de la imagen en la vista del carrito

Trxpxrx committed:
Se agrega la vista del carrito en la version mobile, junto con su hoja de estilo y navbar

Trxpxrx committed:
Se agregaron las versiones responsive del carrito

Trxpxrx committed:
Se agregaron correcciones a la parte del product detail y las versiones responsivas

Trxpxrx committed:
se agrego el login junto con su hoja de estilos

Trxpxrx committed:
Cambios esteticos en el carrito

Trxpxrx committed:
Correcciones de diseño en el detallado del producto

miguelmballeza committed:
Cambios significativos, agregado de páginas completas.

Página de Main, de Registro y de Inicio de Sesión fueron modificadas en
su totalidad para la homogeneidad, los estilos de cada una de las
páginas ya estan relacionados a través de link, solo hace falta corregir
una parte del navbar del "seeker", pero de ahí en fuera todo esta
terminado de manera adaptativa para todas las resoluciones ofrecidas :
576, 768, 992, 1200 y en algunos casos 1500 px.

Trxpxrx committed:
se agrego el nuevo navbar

Trxpxrx committed:
se agrego el footer en las secciones faltantes y se hicieron correcciones

fragmnt02 commented on .gitignore:
```diff
@@ -0,0 +1 @@
+node_modules
```
como tu node_modules está dentro de `src/node_modules` agrega esta ruta al `.gitignore` y borra la carpeta del repo para no tener cientos de archivos extra que no necesitamos.

miguelmballeza replied:
Se agrego en el .gitignore dentro de src, y ya esta listo Pako, gracias.

fragmnt02 commented on src/public/css/normalize.css:
```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```
No deberían usar ninguna librería que les ayude en el css. Borrenlo por favor.

miguelmballeza replied:
También ya fue borrada Pako, espero el Merge por tu parte, gracias!

miguelmballeza committed:
Borrando carpeta de FrameWork Express.

Para no tener archivos inecesarios en el repositorio del proyecto, lo mejor es borrar esta carpeta que hace referencia a el Framework Express.

miguelmballeza committed:
Se agrego el archivo de .gitignore

Se inserto el archivo de .gitignore dentro de la carpeta de "src"
especificando que la carpeta correspondiente a express, la de
"node_modules", no tenia que ser considerada para su subida al
repositorio remoto en GitHub, por eficiencia.

miguelmballeza committed:
Normalize.css y .gitignore

Borrado de normalize.css de src/public/css, por indicaciones de asesor,
y además el borrado del archivo .gitignore desde la raiz, fue incluido
de mejor forma en la carpeta src para que solo se hiciera referencia a
la carpeta de node_modules correspondientes a la instalación de Express.

Please enter the commit message for your changes. Lines starting

miguelmballeza committed:
Borrado de .gitignore desde la raíz
